packages/logs/src/domain/configuration.ts

```diff
@@ -28,11 +39,30 @@ export function validateAndBuildLogsConfiguration(
     return
   }
 
+  let forwardConsoleLogs: StatusType[] = []
+  if (isExperimentalFeatureEnabled('forward-logs')) {
+    if (
+      initConfiguration.forwardConsoleLogs !== undefined &&
+      initConfiguration.forwardConsoleLogs !== 'all' &&
+      !Array.isArray(initConfiguration.forwardConsoleLogs)
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏it could be nice to ensure that only allowed api are instrumented.